client/flutter/lib/components/question_index.dart

```diff
@@ -0,0 +1,114 @@
+import 'package:WHOFlutter/components/question_data.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_html/flutter_html.dart';
+import 'package:url_launcher/url_launcher.dart';
+import 'package:html/dom.dart' as dom;
+
+class QuestionIndexPage extends StatefulWidget {
+  @override
+  _QuestionIndexPageState createState() => _QuestionIndexPageState();
+}
+
+class _QuestionIndexPageState extends State<QuestionIndexPage> {
+  List<QuestionItem> _questions;
+
+  @override
+  void initState() {
+    super.initState();
+    initStateAsync();
+  }
+
+  void initStateAsync() async {
+    // Fetch the dynamic query data. This is a placeholder.
+    _questions = await QuestionData.questions();
+    setState(() { });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(body: _buildPage());
+  }
+
+  Widget _buildPage() {
+    var items = (_questions ?? []).map(_buildQuestion).toList();
+    return Material(
+      color: Colors.grey.shade200,
+      child: CustomScrollView(
+        slivers: [
+          SliverAppBar(
+            backgroundColor: Colors.white,
+            flexibleSpace: FlexibleSpaceBar(
+                background: SafeArea(
+              child: Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                  crossAxisAlignment: CrossAxisAlignment.center,
+                  children: <Widget>[
+                    IconButton(
```

Comment:
Is this button necessary?

Normally it autogenerates a back button

Reply:
Flutter wasn't giving us one with this header arrangement... Perhaps I am missing something.

Reply:
You were right!  I fixed this.